api/data_test.go

```diff
@@ -75,7 +153,7 @@ func TestUpdateExistingCacheTime(t *testing.T) {
 				return nil
 			},
 		}
-		dataStoreAPI := setupAPIWithStore(ctx, dataStoreMock)
+		dataStoreAPI := setupAPINoAuthWithStore(ctx, dataStoreMock)
```

Comment:
How come we're doing this? This does not represent how the application actually works, shouldn't we be passing a token with the request, like it happens in reality? Or is that so complicated that this is how it's done in other repositories where authentication is involved?

Reply:
It looks like it should be possible to test some of the auth features through component testing, although I don't know how easy it would be: https://github.com/ONSdigital/dp-dataset-api/blob/e066b54e049f4695b82b1164eeeea6b74c012f3d/features/access.feature#L9